language/changes/7-packages.md

```diff
@@ -0,0 +1,323 @@
+maps, generated documentation, and ABIs. The Move package system allows
+programmers to easily define a package, parametrize the package by named
+addresses, use the package in their Move code and instantiate the named
+addresses, coordinates the build process, and provides a common interface for
+interacting with compiled artifacts generated from the package.
+
+## Motivation
+
+Being able to re-use and manage code is a crucial activity when developing
+code. However until now, when writing Move you had no easy options of being
+able to package Move code into reusable chunks that could be used by others.
+There was no common compiled object model or on-disk representation of compiled
+Move code and its associated artifacts, e.g., source maps and generated
+documentation. And finally, there was no way of easily managing named addresses
+that spanned "logical groupings" of Move code.
+
+The Move package system aims to make these actions easier by providing:
+* A way for programmers to easily declare logical packages of Move code.
+* A way for programmers to easily import and use those packages.
+* A way for programmers to easily compile and generate associated compilation
+  artifacts from those packages.
+* A common compiled-object model and access to both compiled bytecode and
+  associated compilation artifacts.
+* A way to manage--expose and instantiate--named addresses from packages of Move code.
+
+## Package Layout and Manifest Syntax
+
+A Move package is laid out as follows on-disk:
+
+```
+a_move_package
+├── Move.toml      (required)
+├── sources        (required)
+├── examples       (optional, test & dev mode)
+├── scripts        (optional)
+├── doc_templates  (optional)
+└── tests          (optional, test mode)
+```
+
+The directories marked `required` _must_ be present in order for the directory
+to be considered a Move package and to be compiled. Optional directories can
+be present, and if so will be included in the compilation process. Depending on
+the mode that the package is built with (`test` or `dev`), the `tests` and
+`examples` directories will be included as well.
+
+The `sources` directory can contain both Move modules and Move scripts (both
+transaction scripts, and modules containing script functions). The `examples`
+directory can hold additional code to be used only for development and/or
+tutorial purposes that will not be included when compiled outside `test` or
+`dev` mode.
+
+Move code in the `scripts` directory will be included if present.
+Documentation will be built using any [documentation
+templates](../move-prover/doc/user/docgen.md) present in the `doc_templates`
+directory.
+
+### Move.toml
+
+The Move package manifest is defined within the `Move.toml` file and has the
+following syntax. Optional fields and sections are marked with `*`, `+` denotes
+one or more elements:
+
+```
+[package]                        # Declares information about this package
```

Comment:
I would drop the comment on this. It confused me at first because I thought you were saying that the "package" keyword was supposed to be replaced by a description of the package. Otherwise, you could add more detail to explain that the file is divided into several sections each with a header to identify the type of the section.